src/openvr_plugin/driver_psmoveservice.cpp

```diff
@@ -1563,8 +1613,27 @@ PSMoveQuaternion ExtractYawQuaternion(const PSMoveQuaternion &q)
             0.f, sinf(half_yaw), 0.f); // (x, y, z) = sin(theta/2)*axis, where axis = (0, 1, 0)
 
     return yaw_quaternion;
+}
+
+
+void CPSMoveControllerLatest::GetMetersPosInControllerRotSpace(PSMoveFloatVector3* pOutPosition)
+{
+	const ClientPSMoveView &view = m_controller_view->GetPSMoveView();
```

Comment:
This assumes that the controller view is a PSMove controller (rather than a dual shock 4 or navi controller). If the controller view is one of the other types this GetPSMoveView() call will assert. Instead we should use ClientControllerView::GetPosition() and ClientControllerView::GetOrientation(), which does the switch statement based on controller type. Since the DS4 support is in, but not officially supported, and you have to opt into new feature this doesn't need to be fixed yet. But it's something to keep in mind for the near term.